openvalidators/forward.py

```diff
@@ -285,13 +285,14 @@ async def forward(self):
     best_followup = followup_completions[followup_rewards.argmax(dim=0)].strip()
 
     # Prompt-based scoring via network. Prohibits self-scoring.
-    (
-        followup_scorings,
-        followup_scoring_uids,
-        followup_scoring_completions,
-        followup_scoring_values,
-    ) = await scoring_completions(self, bootstrap_prompt, followup_scoring_template, followup_responses, followup_uids)
-    best_followup_scoring = followup_completions[followup_scorings.argmax(dim=0)].strip()
+    if self.config.neuron.outsource_scoring:
```

Comment:
For the sake of brevity, what if we return a dict object instead of a tuple? This will also be cleaner to unpack into the event

Reply:
Yes, good idea, latest commits have it.